src/quo2/components/banners/banner/view.cljs

```diff
@@ -3,22 +3,29 @@
             [quo2.components.counter.counter :as counter]
             [quo2.components.icon :as icons]
             [quo2.components.markdown.text :as text]
+            [quo2.foundations.colors :as colors]
             [react-native.core :as rn]))
 
 (defn banner
   [{:keys [show-pin? latest-pin-text pins-count on-press]}]
-  [rn/touchable-opacity
-   {:accessibility-label :pinned-banner
-    :style               style/container
-    :active-opacity      1
-    :on-press            on-press}
-   (when show-pin? [icons/icon :i/pin {:size 20}])
-   [text/text
-    {:number-of-lines 1
-     :size            :paragraph-2
-     :style           {:margin-left 10 :margin-right 50}}
-    latest-pin-text]
-   [rn/view
-    {:accessibility-label :pins-count
-     :style               style/counter}
-    (when (pos? pins-count) [counter/counter {:type :secondary} pins-count])]])
\ No newline at end of file
+  (when (pos? pins-count)
+    [rn/touchable-opacity
+     {:accessibility-label :pinned-banner
+      :style               style/container
+      :active-opacity      1
+      :on-press            on-press}
+     (when show-pin?
+       [icons/icon :i/pin
+        {:color (colors/theme-colors
```

Comment:
Looks like unnecessary line breaks.

Reply:
hmm, does our auto formatter not handle stuff like this?

Reply:
and if not can, is it possible to adjust so we don't have to think about issues like this.

Reply:
It handles @J-Son89, but we have configured it to explicitly respect new lines. From a previous PR opened by @yqrashawn, we have found that enabling the rule caused too many questionable format changes.

Reply:
https://github.com/kkinnear/zprint/blob/main/doc/reference.md#respect-nl

Reply:
right but shouldn't make lint-fix adjust this correctly for me? or at least `make lint` can break and tell me that I need to adjust it.

Reply:
ok, I see now. It's too complex to handle and we must do ourselves. Personally would prefer if we can completely automate formatting so it's not a focus of p.r reviews.

Reply:
It's tricky @J-Son89, even the Clojure Style Guide doesn't force certain styles. But just it's clear, I agree with you, we need to keep pushing to improve the format rules (up to a point though).

I think we need to focus in one formatting problem at a time because any rule change can have undesirable ripple effects.

I really suggest that if anyone wants to improve formatting, first change your `zprintrc` file and see if you like the formatted output, check many many files. That's because some configuration changes are good *in theory*, but when you actually reformat they become disaster.

----

Just for context, a couple years ago, there were many heated discussions about auto formatting Clojure a la Prettier.js. We need to understand Lisps have a long history and that s-expressions can be formatted in many different ways, often in very subjective ways. Expressions are not statements as we all know. For example, in Clojure you can have "variable" (binding) declarations almost anywhere, this is not the case in Javascript. The space for subjectivity in Lisps is much much higher. If we decide to format using a very strict set of rules we'll cause more trouble than value in Clojure (as @yqrashawn demonstrated in his past PR to introduce zprint), so we better do one little rule at a time. There's a reason Lisps don't usually force a specific style, because the code end up looking worse in many cases.

Reply:
FYI: @jo-mut, we were talking about format rules just now. For context, see my previous comment in this very thread https://github.com/status-im/status-mobile/pull/14629#discussion_r1056382648